central/sensor/service/pipeline/nodeindex/pipeline.go

```diff
@@ -58,8 +59,10 @@ func (p pipelineImpl) Match(msg *central.MsgFromSensor) bool {
 }
 
 func (p pipelineImpl) Run(ctx context.Context, _ string, msg *central.MsgFromSensor, _ common.MessageInjector) error {
-	if !features.ScannerV4.Enabled() {
-		// If Scanner V4 is disabled do not run this pipeline
+	if !env.NodeIndexEnabled.BooleanSetting() || !features.ScannerV4.Enabled() {
```

Comment:
I am wondering what is the reason we not accessing the `NodeIndexEnabled` through the features package? Is it expected that the indexer is enabled or disabled on the fly (smells like runtime config 😉 )?